migration/migration_blackbox_test.go

```diff
@@ -1181,6 +1182,35 @@ func testMigration99CodebaseCVEScanDefaultFalse(t *testing.T) {
 	require.Nil(t, runSQLscript(sqlDB, "099-codebase-cve-scan-default-false-cleanup.sql"))
 }
 
+// test that the userspace_data table no longer exists - previously
+// used as a temporary solution to get data from tenant jenkins
+func testMigration100DropUserspacedataTable(t *testing.T) {
```

Comment:
Why is this test in this PR?

Reply:
Remember that we talked about this a few days back? This was the renaming issue with the numbering. The test is not actually in there, it is renamed to be consistent with my new test there.